cmd/generate/config/base/config.go

```diff
@@ -75,11 +75,8 @@ func CreateGlobalConfig() config.Config {
 
 				// ----------- JavaScript files -----------
 				// Dependencies and lock files.
-				regexp.MustCompile(`(^|/)node_modules/.*?$`),
-				regexp.MustCompile(`(^|/)package-lock\.json$`),
-				regexp.MustCompile(`(^|/)yarn\.lock$`),
-				regexp.MustCompile(`(^|/)pnpm-lock\.yaml$`),
-				regexp.MustCompile(`(^|/)npm-shrinkwrap\.json$`),
+				regexp.MustCompile(`(^|/)node_modules(/.*?)?$`),
```

Comment:
Due to how `Walk` works this needs to match no trailing `/`, otherwise it will continue to walk every directory below.
```
frontend/node_modules
```